java/connector-node/risingwave-sink-jdbc/src/main/java/com/risingwave/connector/jdbc/JdbcDialect.java

```diff
@@ -63,12 +65,12 @@ void bindUpsertStatement(
      *
      * @return the dialects {@code INSERT INTO} statement.
      */
-    default String getInsertIntoStatement(String tableName, List<String> fieldNames) {
+    default String getInsertIntoStatement(SchemaTableName tableName, List<String> fieldNames) {
         String columns =
                 fieldNames.stream().map(this::quoteIdentifier).collect(Collectors.joining(", "));
         String placeholders = fieldNames.stream().map(f -> "?").collect(Collectors.joining(", "));
         return "INSERT INTO "
-                + quoteIdentifier(tableName)
+                + quoteIdentifier(tableName.getNormalizedTableName())
```

Comment:
It should be `"INSERT INTO " + getNormalizedTableName() ...`. The current code cannot handle databases that quote the schema name and table name separatedly (i.e. `` `schema_name`.`table_name` ``).